src/index.ts

```diff
@@ -1,6 +1,7 @@
 // 🌲☀️🌲
 export { version } from './version';
 export { getFrameHtmlResponse } from './core/getFrameHtmlResponse';
+export { getFarcasterUserAddress } from './farcaster/getFarcasterUserAddress';
```

Comment:
If we want to start doing namespaces, than the `index.ts` to use is `/farcster/index.ts`. 

And then we will have folks doing the long import
```ts
import {  getFarcasterUserAddress } from '@coinbase/onchainkit/farcaster';
```

Reply:
Same for the types.